ts/sagas/startup.ts

```diff
@@ -283,9 +287,22 @@ export function* initializeApplicationSaga(
   // it will handle its own cancelation logic.
   yield* spawn(watchLogoutSaga, backendClient.logout);
 
+  if (zendeskEnabled) {
+    yield* fork(watchZendeskGetSessionSaga, backendClient.getSession);
+  }
+
+  const needRefreshZendeskToken = yield* select(
+    zendeskTokenNeedsRefreshSelector
+  );
   // check if the current session is still valid
   const checkSessionResponse: SagaCallReturnType<typeof checkSession> =
-    yield* call(checkSession, backendClient.getSession);
+    yield* call(
+      checkSession,
+      backendClient.getSession,
+      formatRequestedTokenString(needRefreshZendeskToken)
+    );
+  yield* put(zendeskTokenNeedsRefresh(false));
```

Comment:
Is that needed to "reset" a previous `zendeskToken` refresh handling? We cannot handle the reset outside the startup saga?

Reply:
This logic has been removed.